content/es/docs/zero-code/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Cero código Instrumentación
```

Comment:
@open-telemetry/docs-es-approvers We need to standardize on a name for Zero-code instrumentation. My proposal is "Instrumentación sin código".